cmd/ax/query.go

```diff
@@ -24,6 +24,10 @@ func addQueryFlags(cmd *kingpin.CmdClause) *common.QuerySelectors {
 	cmd.Flag("after", "Results from after").StringVar(&flags.After)
 	cmd.Flag("select", "Fields to select").Short('s').HintAction(selectHintAction).StringsVar(&flags.Select)
 	cmd.Flag("where", "Add a filter").Short('w').HintAction(whereHintAction).StringsVar(&flags.Where)
+	cmd.Flag("one-of", "Add a membership filter (FIELD_NAME:FIELD_VALUE)").HintAction(oneOfHintAction).StringsVar(&flags.OneOf)
```

Comment:
Still the stylistic question, should this not be `where-one-of` to make it more "sentency"